recursion/problem-set-recursion-pt-2.md

```diff
@@ -0,0 +1,290 @@
+# Problem Set: Recursion Pt. 2
+
+## Directions
+
+Complete all questions below. **Solve all questions using recursion**, and not iteration.
+
+## Practice
+
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: code-snippet
+* language: python3.6
+* id: nCv8Qu
+* title: Recursion
+* points: 3
+### !question
+
+Write a function `search` that accepts an unsorted array of integers, `array`, and an integer value to find, `needle`. It returns `True` if `needle` is found in `array`, and `False` otherwise. Make the algorithm recursive.
```

Comment:
suggest a more neutral term for the value we are looking for like `query`. I know this is a reference to a needle in a haystack, but for anyone _not_ familiar with the idiom, the half-idiomatic use might be confusing.
```suggestion
Write a function `search` that accepts an unsorted array of integers, `array`, and an integer value to find, `query`. It returns `True` if `query` is found in `array`, and `False` otherwise. Make the algorithm recursive.
```